Cargo.toml

```diff
@@ -1,25 +1,35 @@
 [package]
-name = "promscale_rs"
-version = "0.3.0"
+name = "promscale"
+version = "0.4.0"
```

Comment:
Yes, can do.

Reply:
Actually coming back to this: I _had_ to bump the version number because otherwise the way that PGX auto-generates and installs the schema conflicted with the existing SQL schema scripts. I chose to bump it by another minor version because I was unsure how backwards compatible we would be able to remain.

I will bump the patch version as you suggested. We can always bump the minor before release if we realise that it's necessary.

Reply:
Yeah let's always bump be patch until release. We can always choose to bump more at realease time but not really go backwards